integrations/hello-world-ng10-ivy/package.json

```diff
@@ -20,17 +20,17 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~10.0.2",
-    "@angular/common": "~10.0.2",
-    "@angular/compiler": "~10.0.2",
-    "@angular/core": "~10.0.2",
-    "@angular/forms": "~10.0.2",
-    "@angular/platform-browser": "~10.0.2",
-    "@angular/platform-browser-dynamic": "~10.0.2",
-    "@angular/router": "~10.0.2",
-    "rxjs": "~6.5.5",
-    "tslib": "^2.0.0",
-    "zone.js": "~0.10.3",
+    "@angular/animations": "10.0.2",
+    "@angular/common": "10.0.2",
+    "@angular/compiler": "10.0.2",
+    "@angular/core": "10.0.2",
+    "@angular/forms": "10.0.2",
+    "@angular/platform-browser": "10.0.2",
+    "@angular/platform-browser-dynamic": "10.0.2",
+    "@angular/router": "10.0.2",
+    "rxjs": "6.5.5",
+    "tslib": "2.4.0",
```

Comment:
@markwhitfeld I updated tslib to the latest versions compatible with the corresponding typescript versions in each package.json.

https://github.com/Microsoft/tslib#installing

> For the latest stable version, run:
> ```
> # TypeScript 3.9.2 or later
> yarn add tslib
> 
> # TypeScript 3.8.4 or earlier
> yarn add tslib@^1
>
># TypeScript 2.3.2 or earlier
>yarn add tslib@1.6.1
>```

From the perspective of maintenance and security, as far as I know, it is good to use the latest versions that are compatible.

Reply:
@markwhitfeld I reverted dependencies upgrade that I made to make tests work after upgrading jest-preset-angular, now integration tests are failing, in particular this one https://github.com/ngxs/store/runs/7186035903?check_suite_focus=true. Possibly that's the reason why I upgraded tslib in integration tests, I don't remember exactly right now since a lot of time has passed. At least dependencies upgrade that I reverted fixes integration tests.

If this two commits are reverted, integration tests should work again:
- https://github.com/ngxs/store/pull/1846/commits/9ce64c71ed233bc371f28156db1273bfdd84390c
- https://github.com/ngxs/store/pull/1846/commits/7a3ffbc59c00c94fe586d93d0862a41043039e83

Reply:
Hmmm... Do you think it could be something to do with the jest preset version that is conflicting with tslib?

Reply:
@markwhitfeld most probably.